plugins/scaffolder-backend/package.json

```diff
@@ -67,6 +67,7 @@
     "lodash": "^4.17.21",
     "luxon": "^3.0.0",
     "morgan": "^1.10.0",
+    "node-abort-controller": "^3.0.1",
```

Comment:
https://nodejs.org/api/globals.html#class-abortcontroller

we require node 16 or higher, so we should actually switch over to using the native ones in node instead of this library.

Reply:
Sure, I did it when it was still node 14, and test was failing due to native abort controller 😅